cfme/configure/configuration/diagnostics_settings.py

```diff
@@ -192,6 +192,8 @@ def update(self, updates, cancel=False, reset=False):
             if self.appliance.version < '5.9':
                 fill_dict['depot_creds']['confirm_password'] = updates.get('password')
         updated = view.fill(fill_dict)
+        if depot_type == 'Red Hat Dropbox':
```

Comment:
Its not clear why this is set specifically for this depot type, please include a comment in-line here as to why its being done.

Also consider setting this at L192 after the existing check that `depot_type != 'Red Hat Dropbox'`. You could set `updated` then, and then at L194 use `updated = view.fill(fill_dict) or updated`, which would default it to True if the fill doesn't return True.